app/common/model/src/main/java/io/syndesis/common/model/integration/IntegrationOverview.java

```diff
@@ -60,6 +61,10 @@ default IntegrationBulletinBoard getBoard() {
 
     Optional<String> getUrl();
 
+    Optional<String> getUrlTo3scale();
```

Comment:
Could it actually be exposed via 'config.json'? Does operator modify configmap, if we change an environment variable after installation? Or will one have to modify both the OPENSHIFT_ULR_TO3SCALE environment variable and config map?

Reply:
It could be added to `config.json`, not sure about operator changing the ConfigMap perhaps @lgarciaaco can answer that. I think it would make sense to make this part of the Syndesis custom resource instead of environment variable...

Reply:
Oh I see that it is part of the custom resource, then the operator can definitely update the `config.json`, not sure if it does currently.